src/components/Person/Person.jsx

```diff
@@ -1 +1,24 @@
-// export const Person = ({ person }) => ();
+function wifeOrHusband(arg) {
+  return arg.sex === 'm' ? 'wife' : 'husband';
+}
+
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+    {person.age !== undefined && (
+      <p className="Person__age">
+        {`I am ${person.age}`}
+      </p>
+    )}
+
+    <p className="Person__partner">
+      {
+        person.isMarried
+          ? `${person.partnerName} is my ${wifeOrHusband(person)}`
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable